src/utils/deployment-for-request.ts

```diff
@@ -2,7 +2,7 @@ import { Config, Deployment } from '../config'
 
 export const deploymentForRequest = (request: Request, config: Config): Deployment | undefined => {
   return config.deployments.find(deployment => {
-    return deployment.routes.find(route => {
+    return deployment.routes.some(route => {
```

Comment:
Why do we change from returning a matching object to returning a boolean? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some